actionpack/test/dispatch/request_test.rb

```diff
@@ -1398,8 +1404,7 @@ class RequestFormData < BaseRequestTest
 class EarlyHintsRequestTest < BaseRequestTest
   def setup
     super
-    @env["rack.early_hints"] = lambda { |links| links }
-    @request = stub_request
+    @request = stub_request({ "rack.early_hints" => lambda { |links| links } })
```

Comment:
`stub_request` already allows passing extra values for `env` via, arguments. No need to load it in `@env`.